### PR TITLE
fix(dynamicWidgets): prevent "ais.dynamicWidgets" attribute showing up

### PR DESCRIPTION
**Pull request by Haroenv**
[FX-613](https://algolia.atlassian.net/browse/FX-613)

### CODE REVIEW

File: e2e/templates.test.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ describe('Templates', () => {
       });
 
       afterAll(() => {
-        execSync(`rm -rf "${temporaryDirectory}"`);
+        // execSync(`rm -rf "${temporaryDirectory}"`);
```
Comment (Haroenv) [inline e2e/templates.test.js:84]:
```suggestion
        execSync(`rm -rf "${temporaryDirectory}"`);
```

**Review by francoischalifour — APPROVED** [review]

File: e2e/__snapshots__/templates.test.js.snap [hunk 2]
```diff
@@ -5748,8 +5767,11 @@ function App() {
         <InstantSearch searchClient={searchClient} indexName=\\"indexName\\">
           <div className=\\"search-panel\\">
             <div className=\\"search-panel__filters\\">
-              <RefinementList attribute=\\"facet1\\" />
-              <RefinementList attribute=\\"facet2\\" />
+              <Configure facets={['*']} maxValuesPerFacet={20} />
```
Comment (tkrugg) [inline e2e/__snapshots__/templates.test.js.snap:5770]: Is this still valid, given it's going to be a different default in a near future?

File: e2e/__snapshots__/templates.test.js.snap [hunk 3]
```diff
@@ -5748,8 +5767,11 @@ function App() {
...
+              <ExperimentalDynamicWidgets fallbackWidget={RefinementList}>
```
Comment (tkrugg) [inline e2e/__snapshots__/templates.test.js.snap:5771]: Will need to be updated after the release as non exprimental

File: e2e/__snapshots__/templates.test.js.snap (hunk 2 above)
Comment (Haroenv) [reply e2e/__snapshots__/templates.test.js.snap:5770]: Yes, and this also isn't relevant to this PR @tkrugg, it was added in #534. When we release a version which supports facets * + maxValuesPerFacet out of the box, I'll change the template and the lowest allowed version

File: e2e/__snapshots__/templates.test.js.snap (hunk 3 above)
Comment (Haroenv) [reply e2e/__snapshots__/templates.test.js.snap:5771]: it's already released as non-experimental, but indeed not yet updated here. Separate PR :)

**Review by tkrugg — APPROVED** [review]

**Pull request merged**